stack.yaml

```diff
@@ -10,4 +10,4 @@ extra-deps:
 - galois-field-1.0.1
 - bitvec-1.0.2.0
 - vector-algorithms-0.8.0.3
-- quiet-0.2@sha256:60bb3cb8dd4a225557351b6d622147a4e541f1564847e05a81621365a155ad6c,1413
+- quiet-0.2
```

Comment:
It gets the same hash in the `stack.yaml.lock` file anyway.

Reply:
I thought that specifying shas in `stack.yaml` was more of a security measure.  I was under the impression that lock file might be edited during an update.

Reply:
But then we'll see that anyway?